assets/blocks/lesson-actions/lesson-actions-block/use-has-quiz.js

```diff
@@ -33,7 +34,12 @@ const useHasQuiz = () => {
 		};
 	}, [] );
 
-	return hasQuiz;
+	const { _quiz_has_questions: quizHasQuestionsMeta } =
```

Comment:
Since we're refactoring it to check the meta, I wonder if we could refactor also the `sensei-quiz-editor-question-count-updated` CustomEvent, removing it, and just setting the meta there too (`assets/js/admin/meta-box-quiz-editor.js` `jQuery.fn.updateQuestionCount`). It could make this simpler by avoiding different checks.

Reply:
Could be a good idea, but it's a bit scary — I'd rather not add calling Gutenberg APIs as a side effect into that script, since it works pretty differently.